src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6009,7 +6059,7 @@ namespace SimAirServingZones {
 
 			// write predefined standard 62.1 report data
 			for ( AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
-				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {
+//				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {  // commented line allows ZoneSum method to report tables to the html file
```

Comment:
@mjwitte wrote: "If it's intended to stay, then the report needs to indicate which method was used."

The only think I can see to do is modify the report name as Standard 62.1 Summary using Zone Sum Method. This seems messy. Any other ideas?